.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.7.4
+2.7.6
```

Comment:
i've seen somewhere when installing gems or somewhere else information that they encourage me to change version of ruby to newest (2.7.6) because of bugfixes, etc, so i updated (as nobody else is working on this project)